src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -853,6 +878,17 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         _renderer->TriggerRedrawAll();
     }
 
+    // Method Description:
+    // - Toggle the cursor on and off when called by the cursor blink timer.
+    // Arguments:
+    // - sender: not used
+    // - e: not used
+    void TermControl::_BlinkCursor(Windows::Foundation::IInspectable const& /* sender */,
+                                   Windows::Foundation::IInspectable const& /* e */)
+    {
+        if (!(_terminal->GetTextBuffer().GetCursor().IsBlinkingAllowed() || _terminal->IsCursorOn()) || _terminal->IsCursorVisible()) _terminal->SetCursorOn(!_terminal->IsCursorOn());

```

Comment:
Line is too long. Also, we require `{}` for the code block following an if statement.

Reply:
What is the preferred line length for the Terminal project?

Reply:
I don't think we have an actual hard number. I personally try to break a line once it's 100 cells wide. I assume clang-format has an option to enforce this so we'll be codifying an actual number here soon.